image-builder/builder-services/src/main/java/io/hyscale/builder/services/docker/impl/DockerRESTClient.java

```diff
@@ -339,14 +346,16 @@ public void onNext(PushResponseItem item) {
         try {
             pushImageCmd.exec(callback).awaitCompletion();
         } catch (RuntimeException e) {
-            handleOutput(buildContext.isVerbose(), pushActivity, Status.FAILED);
+            handleOutput(isVerbose, pushActivity, Status.FAILED);
             throw new HyscaleException(e, ImageBuilderErrorCodes.FAILED_TO_PUSH_IMAGE);
-        } catch (InterruptedException  e) {
+        } catch (InterruptedException e) {
             Thread.currentThread().interrupt();
-            handleOutput(buildContext.isVerbose(), pushActivity, Status.FAILED);
+            handleOutput(isVerbose, pushActivity, Status.FAILED);
             throw new HyscaleException(e, ImageBuilderErrorCodes.FAILED_TO_PUSH_IMAGE);
         }
-        handleOutput(buildContext.isVerbose(), pushActivity, Status.DONE);
+        handleOutput(isVerbose, pushActivity, Status.DONE);
+        String shaSum = sb.toString();
```

Comment:
1.Remove this shaSum String variable. 
2. Rename sb to shaSum.
3. Update return statement as **return StringUtils.isNotBlank(shaSum) ? shaSum.toString() : null;**